### PR TITLE
Add release note instructions

### PR DESCRIPTION
**Pull request by xylix**

### CODE REVIEW

File: development.rst [hunk 1]
```diff
@@ -29,5 +29,9 @@ Making a release
 #. Create a signed tag: :code:`git tag -s v0.7.1`
 #. Push the commit and tag: :code:`git push origin refs/tags/v0.7.1`
 #. Create a release on GitHub
+  - For release notes:
+    - Main repo `git log $PREVIOUS_RELEASE_TAG...master --oneline --decorate >> commit_summary.txt`
```
Comment (johan-bjareholt) [inline development.rst]: Maybe easier to maintain this snippet if it would be added as a script in the main repo?

Comment (xylix) [reply development.rst]: Hm yeah fair point. I'll do a PR for it in the main repo and then update this fork.

Comment (xylix) [reply development.rst]: Updated

Comment (johan-bjareholt) [reply development.rst]: great! :)

**Pull request merged**